make_manifest.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+DOCKER_NS=${DOCKER_NS:-openfaas}
+TAG=${TRAVIS_TAG:-latest}
+NAME=license-check
+
+docker manifest create $DOCKER_NS/$NAME:$TAG \
+  $DOCKER_NS/$NAME:$TAG-darwin \
+  $DOCKER_NS/$NAME:$TAG-x86_64 \
+  $DOCKER_NS/$NAME:$TAG-armhf \
+  $DOCKER_NS/$NAME:$TAG-arm64 \
+  $DOCKER_NS/$NAME:$TAG-windows
+
+docker manifest annotate $DOCKER_NS/$NAME:$TAG --arch arm $DOCKER_NS/$NAME:$TAG-darwin
```

Comment:
What about the two additional IBM architectures we have in the repo?

i.e. s390x and ppc64le

Reply:
What is the arch for that?  same string?

Reply:
Yes, I thought I commented, I think it was below.

Reply:
Just added these